ts/utils/hooks/bottomSheet.tsx

```diff
@@ -168,3 +175,62 @@ export const useIOBottomSheetModal = (
   );
   return { present, dismiss: dismissAll, bottomSheet };
 };
+
+/**
+ * Hook to generate a bottomSheet with a title, snapPoint and a component, that autosizes to the height of its content
+ * @param bottomSheetOptions
+ * @see {BottomSheetOptions}
+ * @param bottomPadding the bottom padding of the bottom sheet, defaul is 0
+ */
+export const useIOBottomSheetAutoresizableModal = (
+  {
+    component,
+    title,
+    snapPoint: originSnap,
+    footer,
+    onDismiss
+  }: BottomSheetOptions,
+  bottomPadding: number = 0
```

Comment:
What do you think if the `bottomPadding` default value is greater than zero using a common value which can be useful everywhere in order to avoid adding it every time we use this hook?